assets/src/animation/effects/zoom/animationProps.js

```diff
@@ -27,25 +27,19 @@ import PropTypes from 'prop-types';
 /**
  * Internal dependencies
  */
-import { FIELD_TYPES } from '../../constants';
+import { FIELD_TYPES, SCALE_DIRECTION } from '../../constants';
 import { AnimationInputPropTypes } from '../types';
 
 export const ZoomEffectInputPropTypes = {
-  zoomFrom: PropTypes.shape(AnimationInputPropTypes),
-  zoomTo: PropTypes.shape(AnimationInputPropTypes),
+  scaleDirection: PropTypes.shape(AnimationInputPropTypes),
 };
 
 export default {
-  zoomFrom: {
-    label: __('From', 'web-stories'),
-    tooltip: __('Valid values range from 0 to 1', 'web-stories'),
-    type: FIELD_TYPES.FLOAT,
-    defaultValue: 0,
-  },
-  zoomTo: {
-    label: __('To', 'web-stories'),
-    tooltip: __('Valid values range from 0 to 1', 'web-stories'),
-    type: FIELD_TYPES.FLOAT,
-    defaultValue: 1,
+  scaleDirection: {
+    label: __('Direction', 'web-stories'),
+    tooltip: __('Valid values are scaleIn or scaleOut', 'web-stories'),
```

Comment:
Let's use placeholders for these values, as they shouldn't be translated.

```suggestion
    tooltip:
      /* translators: 1: scaleIn. 2: scaleOut */
      sprintf(__('Valid values are %1$s or %2$s', 'web-stories'), 'scaleIn', 'scaleOut'),
```